modules/custom/az_core/config/install/filter.format.az_plain_text.yml

```diff
@@ -0,0 +1,13 @@
+langcode: en
+status: true
+dependencies: {  }
+name: 'AZ Plain Text'
+format: az_plain_text
+weight: 0
+filters:
```

Comment:
Is it intended that this format removes the `filter_html_escape` filter normally present in `plain_text`?

It seems like the only filter we'd want to remove would be `filter_url`

Reply:
I tried to use the minimum filters necessary not based on what was used with the plain text format. What does `filter_html_escape` do?

Reply:
`filter_html_escape` is the filter that encodes raw encountered HTML into HTML entities. For example, if that filter is enabled, if the user enters `<script>alert('run some javascript');</script>` into the text area, the `filter_html_escape` filter will cause that to show as plain text. Without that filter enabled, the HTML will be treated as HTML by the browser and the javascript would run.

Most text formats enable either `filter_html` (if they allow some tags), or `filter_html_escape` (if they allow no tags). Text formats do not enable either of the two filters allow raw HTML.

In the case of the Drupal core `plain_text` format, `filter_html_escape` filter is run before `filter_autop` and `filter_url` to avoid eating the html produced by those filters.